client/src/components/cards/Card.svelte

```diff
@@ -0,0 +1,119 @@
+<script>
```

Comment:
can you make a common component without the hardcoded recipe stuff here, it gets all it needs from props. Then you just use it inside recipes without it being hardcoded here?